lib/smart_proxy_pulp_plugin/pulp_disk.rb

```diff
@@ -0,0 +1,21 @@
+module PulpProxy
+  class PulpDisk
+    def initialize(path = PulpProxy::Plugin.settings.pulp_dir)
+      @path = path
+    end
+
+    def df_h
+      `df -h #{@path}`.split("\n").last
```

Comment:
> @lzap do you mean https://github.com/djberg96/sys-filesystem ?

Yeah but let's not introduce new deps, I think executing df is fine
enough.

-- 
Later,
 Lukas #lzap Zapletal